scripts/open_url_in_instance.sh

```diff
@@ -6,13 +6,10 @@ _url="$1"
 _qb_version='0.10.1'
 _proto_version=1
 _ipc_socket="${XDG_RUNTIME_DIR}/qutebrowser/ipc-$(echo -n "$USER" | md5sum | cut -d' ' -f1)"
+_qute_bin="/usr/bin/qutebrowser"
 
-if [[ -e "${_ipc_socket}" ]]; then
-	exec printf '{"args": ["%s"], "target_arg": null, "version": "%s", "protocol_version": %d, "cwd": "%s"}\n' \
-				"${_url}" \
-				"${_qb_version}" \
-				"${_proto_version}" \
-				"${PWD}" | socat - UNIX-CONNECT:"${_ipc_socket}"
-else
-    exec /usr/bin/qutebrowser --backend webengine "$@"
-fi
+exec printf '{"args": ["%s"], "target_arg": null, "version": "%s", "protocol_version": %d, "cwd": "%s"}\n' \
+            "${_url}" \
+            "${_qb_version}" \
+            "${_proto_version}" \
+            "${PWD}" | socat - UNIX-CONNECT:"${_ipc_socket}" 2>/dev/null || $_qute_bin --backend webengine "$@" &
```

Comment:
Please quote `"$_qute_bin"` - it's unlikely to contain spaces, but it could :wink:

Reply:
I'm actually surprised there is not test for this. :wink:

Reply:
There's no test for this script at all, because it's not really part of qutebrowser's core and almost nobody uses it :wink:

Reply:
It's a shame people don't, because it *really* speeds things up. (Maybe people just don't know this script exists/that you can do this via IPC?)

Reply:
While I'm at it, the "--backend webengine" isn't really needed in there, is it?

Reply:
Yep, not anymore. And while you're at it, you could also bump the version in the script.